core/cap-0033.md

```diff
@@ -375,7 +391,12 @@ If op.sponsoredID = op.sourceAccount
 The behavior of `SponsorFutureReservesOp op` is:
 ```
 If an account is sponsoring future reserves for op.sponsoredID
-    Fail with SPONSOR_FUTURE_RESERVES_ALREADY_EXISTS
+    Fail with SPONSOR_FUTURE_RESERVES_ALREADY_SPONSORED
```

Comment:
This refers to already being sponsored just in the ephemeral sense inside this transaction, is that right?